demos/bookinfo/test.py

```diff
@@ -14,11 +14,11 @@
 
 ##############################################################################################
 # Name: Bookinfo
-# Info: support bookinfo application with gwctl inside the clusters
+# Info: support bookinfo application inside the clusters
 #       In this we create three kind clusters
-#       1) cluster1- contain gw, gwctl,product and details microservices (bookinfo services)
-#       2) cluster2- contain gw, gwctl, review-v2 and rating microservices (bookinfo services)
-#       3) cluster3- contain gw, gwctl, review-v3 and rating microservices (bookinfo services)
+#       1) cluster1- contain gw, product and details microservices (bookinfo services)
```

Comment:
nit: unrelated to this change but we should probably be consistent in using lower case or upper case `gw`/`GW` in comments and elsewhere (e.g., here in python vs demos/speedtest/kind/README.md)